sites/fast-website/src/app/css/style.css

```diff
@@ -187,20 +187,39 @@ fast-divider::after {
 }
 
 .section-badge {
```

Comment:
@SamanthaAO a PR recently went in to support a CSS part so you can target the badge there.

To be clear on my previous comment, the badge supports attributes to map colors to it.

```css
fast-badge {
    --badge-fill-foo: var(--background-color);
    --badge-color-foo: var(--accent-foreground-rest);
}
```

```html
<fast-badge fill="foo" color="foo">Badge</fast-badge>
```

Reply:
@chrisdholt yes, that was my understanding for your previous comment. However, once the fill is applied to the internal div through the variable assigned to fast-badge it means that I must also apply the padding in the same place that the fill is assigned. I will be able to do this now with part thanks to the other PR. Thank you for letting me know.